R/constants.R

```diff
@@ -0,0 +1,12 @@
+COMMON.ALWAYS.AVAILABLE.STOCK.GENOMES <-
```

Comment:
Please change to sth simpler without the dots, e.g. `cas_genomes` and `basic_genomes`.

Reply:
done